lib/modules/subredditManager.js

```diff
@@ -1072,18 +1074,45 @@ async function getSubreddits() {
 	} while (after);
 
 	// Remove duplicate subreddits
-	mySubreddits = _.uniqBy(mySubreddits, sr => sr.display_name);
+	mySubreddits = organizeMyData(mySubreddits, sr => sr.display_name);
 
-	mySubreddits.sort((a, b) => {
-		const adisp = a.display_name.toLowerCase();
-		const bdisp = b.display_name.toLowerCase();
+	gettingSubreddits = false;
+	populateSubredditDropdown();
+}
+
+let myMultis = [];
+async function getMultis() {
+	if (gettingMultis) return;
+	gettingMultis = true;
+
+	myMultis = [];
+	const multis = await ajax({
+		url: `/api/multi/user/${loggedInUser()}`,
+		type: 'json',
+		cacheFor: DAY,
+	});
+
+	if (multis) {
+		myMultis = organizeMyData(multis, multi => multi.data.display_name);
+	}
+
+	gettingMultis = false;
+}
+
+function organizeMyData(data, dispFn) {
```

Comment:
This can be left in `getSubreddits`: multis don't need to be sorted since we only care about their number; they don't need to be deduped since we're only making one API call